Explorer/Assets/DCL/Profiles/ProfileSDKSubProduct.cs

```diff
@@ -0,0 +1,64 @@
+﻿using CommunicationData.URLHelpers;
+using DCL.AvatarRendering.Wearables;
+using DCL.ECSComponents;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.Profiles
+{
+    /// <summary>
+    ///     Contains data related to the Player needed for SDK only
+    /// </summary>
+    public class ProfileSDKSubProduct : IDirtyMarker
+    {
+        public AvatarSubProduct Avatar { get; } = new ();
+
+        public bool IsDirty { get; set; }
+
+        public string? Name { get; internal set; }
+
+        public string? UserId { get; private set; }
+
+        public bool HasConnectedWeb3 { get; private set; }
+
+        public void OverrideWith(Profile profile)
+        {
+            Avatar.OverrideWith(profile.Avatar);
+
+            Name = profile.Name;
+            UserId = profile.UserId;
+            HasConnectedWeb3 = profile.HasConnectedWeb3;
+
+            IsDirty = true;
+        }
+
+        public class AvatarSubProduct
```

Comment:
Since its a nested class, you could just name it `Avatar`

Reply:
When we search symbols the full name could me omitted that leads to confusion so I will rename it to `SDKAvatar` similarly to `SDKProfile`